pkg/asset/manifests/openshift.go

```diff
@@ -76,10 +83,15 @@ func (o *Openshift) Generate(dependencies asset.Parents) error {
 			},
 		}
 	case "openstack":
-		clouds, err := clientconfig.LoadCloudsYAML()
+		opts := new(clientconfig.ClientOpts)
+		cloud, err := clientconfig.GetCloudFromYAML(opts)
```

Comment:
Do we also need to modify https://github.com/openshift/installer/blob/master/pkg/asset/installconfig/openstack/openstack.go#L24 in a similar way?

Reply:
No, this is only relevant for the marshal step. The install config step should work just fine.

Reply:
Ack thanks for clarifying